### DISCUSSION TITLE
Table resizable columns dynamically

### DISCUSSION THREAD

**Discussion by Kyojuro27** [ant-design-blazor/ant-design-blazor#2802] [Ideas]
Hi,

I would like to know if an update is on your roadmap about the possibility of resizing dynamically a column with a drag manipulation for the Table component. It's a very useful feature and I saw that Ant-Design for React had a component Table with this kind of feature.

**Comment by ElderJames** [comment]
Hi @Kyojuro27 , we agree that this is a great feature, but we haven't found an implementation yet. Can you help us?
